# Proximity sensor state message
bool state                  # is the sensor energized?
bool normally_open          # true if the sensor is energized when a target is detected
